### PR TITLE
Restructure

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Restructured the wiki. This should be the first step of restructuring and we should merge it asap. So we can all together build on top of it.

Some open questions:

- [ ] Do we keep the Use IOTA section only for the wallet content? Because the other content isn't ready yet
- [ ] We may need to look into introduction pages for many areas, but mainly for the Navbar
    - The solution for this is probably to use dropdown for now

## Type of change

- Documentation Enhancement

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

File: content/about/sidebars.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      collapsed: true,
+      items: [
+        'media-library/infographics',
+        'media-library/project-presentations',
+        'media-library/videos',
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Future of IOTA',
```
Comment (lucas-tortora) [inline content/about/sidebars.ts:37]: I thought we were removing this section

File: content/about/sidebars.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        'future/shimmer',
+        'future/assembly',
+        'future/role-of-iota',
+        'future/dust-protection',
+        'future/nfts',
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Use Cases',
```
Comment (lucas-tortora) [inline content/about/sidebars.ts:65]: This may work for the `Use IOTA` section

File: content/community/sidebars.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+            href: '/IOTA-2.0-Research-Specifications/Preface'
+          },
+          {
+            type: 'link',
+            label: 'GoShimmer',
+            href: '/goshimmer/welcome'
+          },
+          'research/iota-devnet-wallet',
+        ],
+      },
```
Comment (lucas-tortora) [inline content/community/sidebars.ts:40]: This feels a bit out of place here, though it doesn't really fit in `develop` either

File: content/develop/sidebars.ts [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+                type: 'link',
+                href: '/hornet/welcome',
+            },
+            {
+                label: 'Bee',
+                type: 'link',
+                href: '/bee/welcome',
+            },
+        ],
+      },
```
Comment (lucas-tortora) [inline content/develop/sidebars.ts:81]: Perhaps we can combine these 2? Maybe even add INX plugins as a subcategory

File: content/develop/sidebars.ts [hunk 5]
```diff
@@ -0,0 +1,172 @@
+module.exports = {
+    develop: [
+      {
+        label: 'Smart Contracts',
+        type: 'link',
+        href: '/smart-contracts/overview'
+      },
```
Comment (lucas-tortora) [inline content/develop/sidebars.ts:7]: Could we combine this and the WASP entry?

File: content/about/sidebars.ts (hunk 2 above)
Comment (Dr-Electron) [reply content/about/sidebars.ts:65]: Ok. Let's move it then. We may need to check all this docs again tho. One day we need to have a call where we go over the whole nativ wiki content. 😅

File: content/develop/sidebars.ts (hunk 4 above)
Comment (Dr-Electron) [reply content/develop/sidebars.ts:81]: I just followed the miro board. I actually like the current structure better :trollface: 
Maybe something like:
```
Node Software/
├── Hornet
├── Bee
├── INX Plugins/
│   ├── Dashboard
│   ├── Spammer
│   ├── Poi
│   └── ...
└── Extensions/
    ├── Wasp
    └── Chronicle
```

We could also add SC and Permanode to the last two. But we at least need some introduction here somewhere. Nobody knows what Wasp and Chronicle ist 😅

File: content/develop/sidebars.ts (hunk 5 above)
Comment (Dr-Electron) [reply content/develop/sidebars.ts:7]: So what if we add a Category
And linke to different section in the SC docs for now?
Something like:
```
Smart Contracts/
├── Documentation
├── Node (WASP)
└── Schema tool or Tools in general 🤔 
```

**Comment by Dr-Electron** [comment]
Btw I will remove the media section. Will be hosted somewhere else. But looking at the video page, it may make sense to move it in the community section? 
Wdyt?

**Comment by huhn511** [comment]
> Btw I will remove the media section. Will be hosted somewhere else. But looking at the video page, it may make sense to move it in the community section? Wdyt?

The video page list looks like an "awesome list", which is a very nice community/ecosystem info list. The Community section would be the perfect place for it. 

Should we change it to an "awesome list" and open it for more topics? Examples: https://github.com/topics/awesome-lists

**Comment by Dr-Electron** [comment]
> > Btw I will remove the media section. Will be hosted somewhere else. But looking at the video page, it may make sense to move it in the community section? Wdyt?
> 
> The video page list looks like an "awesome list", which is a very nice community/ecosystem info list. The Community section would be the perfect place for it.
> 
> Should we change it to an "awesome list" and open it for more topics? Examples: https://github.com/topics/awesome-lists

I love awesome lists (although I never finish them :trollface: https://github.com/Dr-Electron/Awesome-DevX). So sure 👍

**Comment by huhn511** [comment]
> > > Btw I will remove the media section. Will be hosted somewhere else. But looking at the video page, it may make sense to move it in the community section? Wdyt?
> > 
> > 
> > The video page list looks like an "awesome list", which is a very nice community/ecosystem info list. The Community section would be the perfect place for it.
> > Should we change it to an "awesome list" and open it for more topics? Examples: https://github.com/topics/awesome-lists
> 
> I love awesome lists (although I never finish them :trollface: https://github.com/Dr-Electron/Awesome-DevX). So sure 👍

These will never be finished 🙈 https://github.com/iota-community/awesome-iota

File: content/community/sidebars.ts (hunk 3 above)
Comment (Dr-Electron) [reply content/community/sidebars.ts:40]: UX suggested moving it to `Learn`. Although I would still prefer having it in the Navbar (😢), I can live with that 👍

**Comment by Dr-Electron** [comment]
Closed in favour of https://github.com/iota-wiki/iota-wiki/pull/754

**Pull request closed without merging**